interface/reports/insurance.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Financial Report initial page
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once '../../vendor/autoload.php';
+require_once '../globals.php';
+
+use OpenEMR\Finance\Reports\FinancialSummaryByInsuranceController;
```

Comment:
this will have to match your entry in `src` so believe should change it to be `src\FinancialReports...`

Reply:
Right, I am writing this in 5.0.1(7) so the directories are different as you pointed out. I will make this adjustment. Thanks for the feedback.

Reply:
I have 5.0.2.  I will port the changes over and get back you.

Reply:
Ok, I have it working in 5.0.2. What should the name of the repo be? I will kill this one and make a new one. Proof_of_concept_Twig_OeMR?
@stephenwaite